packages/opencensus-core/src/trace/model/types.ts

```diff
@@ -30,6 +30,17 @@ export interface Attributes {
   [attributeKey: string]: string|number|boolean;
 }
 
+/**
+ * The status of a Span by providing a standard code in conjunction
+ * with an optional descriptive message.
+ */
+export interface Status {
+  /** The status code. */
```

Comment:
Could you add in the comment that this status code should correspond with a specific set of OpenCensus trace status codes and that HTTP status codes should be appropriately converted per the specs at  https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/HTTP.md#mapping-from-http-status-codes-to-trace-status-codes?

Reply:
Done as per https://github.com/census-instrumentation/opencensus-node/pull/328#discussion_r253985285.